apps/nextjs/src/components/AppComponents/Chat/drop-down-section/chat-section.tsx

```diff
@@ -0,0 +1,39 @@
+import { sectionToMarkdown } from "@oakai/aila/src/protocol/sectionToMarkdown";
+import { humanizeCamelCaseString } from "@oakai/core/src/utils/humanizeCamelCaseString";
+import { OakFlex } from "@oaknational/oak-components";
+import { lessonSectionTitlesAndMiniDescriptions } from "data/lessonSectionTitlesAndMiniDescriptions";
+
+import { MemoizedReactMarkdownWithStyles } from "../markdown";
+import FlagButton from "./flag-button";
+import ModifyButton from "./modify-button";
+
+const ChatSection = ({
+  objectKey,
+  value,
+}: {
+  objectKey: string;
```

Comment:
Could this be a keyof `lessonSectionTitlesAndMiniDescriptions`? Or does that complicate things upstream?

Reply:
Will take a look!

Reply:
Looks like this is like this because it's like this anywhere. Def agree good to tighten the types up.

I think a post launch change will largely involve securing lesson plan data integrity throughout the stack (incl relational data in db for it).

Have changed the value type to `unknown` however!